src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -227,16 +227,16 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   }
 
   /**
-   * Formats the org or person name for display.
-   * @param filing The filing body which contains the name/title.
-   * @returns The formatted org/person name.
+   * Returns the org or person full name for display.
+   * @param orgPerson the org/person object
+   * @returns the formatted name
    */
-  private formatName (filing: any): string {
-    if (filing?.officer?.orgName) {
-      return filing.officer.orgName
+  private formatName (orgPerson: OrgPersonIF): string {
+    if (orgPerson?.officer?.orgName) {
+      return orgPerson.officer.orgName
     }
-    if (filing?.officer) {
-      return `${filing.officer.firstName} ${filing.officer.middleName || ''} ${filing.officer.lastName}`
+    if (orgPerson?.officer) {
+      return this.formatFullName(orgPerson.officer)
```

Comment:
Now using common method that doesn't have 2 spaces between first and last name when there is no middle name.